ibis/backends/duckdb/tests/test_geospatial.py

```diff
@@ -210,3 +212,14 @@ def test_geospatial_convert(geotable, gdf):
     gtm.assert_geoseries_equal(
         geo_ll.to_pandas(), gdf_ll, check_less_precise=True, check_crs=False
     )
+
+
+def test_create_table_geospatial_types(geotable, con):
+    name = ibis.util.gen_name("geotable")
+
+    # con = ibis.get_backend(geotable)
+
+    t = con.create_table(name, geotable)
```

Comment:
```suggestion
    t = con.create_table(name, geotable, temp=True)
```

Just so it gets dropped automatically when the test is finished.